fastcdr.repos

```diff
@@ -3,3 +3,7 @@ repositories:
         type: git
         url: https://github.com/eProsima/Fast-CDR.git
         version: master
+    googletest-distribution:
+        type: git
+        url: https://github.com/google/googletest.git
+        version: release-1.11.0
```

Comment:
This probably should not be included in the PR

Reply:
Done in 7a6d9ad